geomesa-accumulo/geomesa-accumulo-process/src/main/scala/org/locationtech/geomesa/accumulo/process/ArrowConversionProcess.scala

```diff
@@ -24,6 +24,7 @@ import org.locationtech.geomesa.utils.collection.SelfClosingIterator
 import org.opengis.feature.Feature
```

Comment:
these processes should go into the geomesa-process module right? so that they can be used by accumulo, hbase, etc eventually.